media/src/polls/Polls.svelte

```diff
@@ -130,18 +130,18 @@
                                     <th scope="row">{poll.id}</th>
                                     <td>{poll.prompt}</td>
                                     <td>{POLL_TYPES[poll.type]}</td>
-                                    <td>
+                                    <td class="row">
```

Comment:
I don't think Bootstrap's `row` class is really meant to be used inside a table cell like this, though I suppose it can work.

This change makes the buttons take up a lot of space, and less table rows will fit on the page:
![Screenshot_2023-03-01_09-53-28](https://user-images.githubusercontent.com/59292/222176052-2c701d5e-2c29-4733-b728-76d1a95ab820.png)

I liked things better how they were before, but this is really Marc's realm - we can discuss with the rest of the team.